src/UserMovieCard/UserMovieCard.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+// import './MovieCard.css';
+
+const UserMovieCard = ({ title, averageRating, poster, id, key, usersRatings }) => {
+	const foundRating = usersRatings.find(rating => rating.movie_id === id);
+
+	return (
+		<section className='movie-card' id={id}>
+			<h3>{title}</h3>
+			<p>Rating: {foundRating ? foundRating.rating : averageRating}/10</p>
```

Comment:
I like the ternary, should we have a specific notification to let us know we are seeing the users rating or average rating?